### PR TITLE
Update photon image

### PR DESCRIPTION
**Pull request by gerrit-photon**
Update photon image

### CODE REVIEW

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 4ac02a3..5d96957 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,4 +1,6 @@
 photon:1.0-20181017 @ amd64
+photon:3.0-20181030 @ amd64
+photon:3.0-20181030 @ arm64v8
 photon:dev-20180904 @ amd64
 photon:dev-20180904 @ arm64v8
 photon:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index b6fba4c..843c4bd 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -2,6 +2,8 @@ photon:1.0
 photon:1.0-20181017
 photon:2.0
 photon:2.0-20181017
+photon:3.0
+photon:3.0-20181030
 photon:dev
 photon:dev-20180904
 photon:latest
diff --git a/photon_dev-20180904/Dockerfile b/photon_3.0-20181030/Dockerfile
similarity index 34%
copy from photon_dev-20180904/Dockerfile
copy to photon_3.0-20181030/Dockerfile
index f2888d8..b4bd5f4 100644
--- a/photon_dev-20180904/Dockerfile
+++ b/photon_3.0-20181030/Dockerfile
@@ -1,8 +1,8 @@
 FROM scratch
-ADD photon-rootfs-2.0-7ca95e0.tar.bz2 /
+ADD photon-rootfs-3.0-5e45dc9.tar.gz /
 
-LABEL name="Photon OS Dev Base Image" \
+LABEL name="Photon OS 3.0 Base Image" \
     vendor="VMware" \
-    build-date="20180901"
+    build-date="20181030"
 
 CMD ["/bin/bash"]
diff --git a/photon_dev-20180904/photon-rootfs-2.0-7ca95e0.tar.bz2 b/photon_3.0-20181030/photon-rootfs-3.0-5e45dc9.tar.gz
similarity index 58%
copy from photon_dev-20180904/photon-rootfs-2.0-7ca95e0.tar.bz2
copy to photon_3.0-20181030/photon-rootfs-3.0-5e45dc9.tar.gz
index 1105a4c..c3d893e 100644
Binary files a/photon_dev-20180904/photon-rootfs-2.0-7ca95e0.tar.bz2 and b/photon_3.0-20181030/photon-rootfs-3.0-5e45dc9.tar.gz differ
diff --git a/photon_latest/photon-rootfs-2.0-045c453.tar.bz2  'tar -t' b/photon_3.0-20181030/photon-rootfs-3.0-5e45dc9.tar.gz  'tar -t'
similarity index 96%
copy from photon_latest/photon-rootfs-2.0-045c453.tar.bz2  'tar -t'
copy to photon_3.0-20181030/photon-rootfs-3.0-5e45dc9.tar.gz  'tar -t'
index c8c2efa..3ed82b9 100644
--- a/photon_latest/photon-rootfs-2.0-045c453.tar.bz2  'tar -t'	
+++ b/photon_3.0-20181030/photon-rootfs-3.0-5e45dc9.tar.gz  'tar -t'	
@@ -542,6 +542,7 @@ usr/bin/sleep
 usr/bin/sort
 usr/bin/split
 usr/bin/stat
+usr/bin/stty
 usr/bin/su
 usr/bin/sync
 usr/bin/tac
@@ -651,27 +652,24 @@ usr/lib/krb5/plugins/preauth/pkinit.so
 usr/lib/krb5/plugins/preauth/test.so
 usr/lib/krb5/plugins/tls/
 usr/lib/krb5/plugins/tls/k5tls.so
-usr/lib/ld-2.26.so
+usr/lib/ld-2.28.so
 usr/lib/ld-linux-x86-64.so.2
-usr/lib/libanl-2.26.so
+usr/lib/libanl-2.28.so
 usr/lib/libanl.so
 usr/lib/libanl.so.1
-usr/lib/libBrokenLocale-2.26.so
+usr/lib/libBrokenLocale-2.28.so
 usr/lib/libBrokenLocale.so
 usr/lib/libBrokenLocale.so.1
 usr/lib/libbz2.so.1
 usr/lib/libbz2.so.1.0
 usr/lib/libbz2.so.1.0.6
-usr/lib/libc-2.26.so
+usr/lib/libc-2.28.so
 usr/lib/libcap.so.2
 usr/lib/libcap.so.2.25
-usr/lib/libcidn-2.26.so
-usr/lib/libcidn.so
-usr/lib/libcidn.so.1
 usr/lib/libcom_err.so
 usr/lib/libcom_err.so.2
 usr/lib/libcom_err.so.2.1
-usr/lib/libcrypt-2.26.so
+usr/lib/libcrypt-2.28.so
 usr/lib/libcrypto.so.1.0.0
 usr/lib/libcrypto.so.1.0.2
 usr/lib/libcrypt.so
@@ -683,15 +681,15 @@ usr/lib/libcurl.so.4.5.0
 usr/lib/libdb-5.3.so
 usr/lib/libdb-5.so
 usr/lib/libdb.so
-usr/lib/libdl-2.26.so
+usr/lib/libdl-2.28.so
 usr/lib/libdl.so
 usr/lib/libdl.so.2
-usr/lib/libelf-0.169.so
+usr/lib/libelf-0.174.so
 usr/lib/libelf.so.1
 usr/lib/libexpat.so.1
-usr/lib/libexpat.so.1.6.6
+usr/lib/libexpat.so.1.6.8
 usr/lib/libformw.so.6
-usr/lib/libformw.so.6.0
+usr/lib/libformw.so.6.1
 usr/lib/libfreeblpriv3.so
 usr/lib/libgcc_s.so.1
 usr/lib/libgssapi_krb5.so
@@ -700,7 +698,6 @@ usr/lib/libgssapi_krb5.so.2.2
 usr/lib/libgssrpc.so
 usr/lib/libgssrpc.so.4
 usr/lib/libgssrpc.so.4.2
-usr/lib/libhawkey.so.2
 usr/lib/libhistory.so.7
 usr/lib/libhistory.so.7.0
 usr/lib/libk5crypto.so
@@ -727,65 +724,57 @@ usr/lib/libkrb5support.so
 usr/lib/libkrb5support.so.0
 usr/lib/libkrb5support.so.0.1
 usr/lib/liblzma.so.5
-usr/lib/liblzma.so.5.2.3
-usr/lib/libm-2.26.so
+usr/lib/liblzma.so.5.2.4
+usr/lib/libm-2.28.so
 usr/lib/libmemusage.so
 usr/lib/libmenuw.so.6
-usr/lib/libmenuw.so.6.0
+usr/lib/libmenuw.so.6.1
 usr/lib/libm.so
 usr/lib/libm.so.6
-usr/lib/libmvec-2.26.so
+usr/lib/libmvec-2.28.so
 usr/lib/libmvec.so
 usr/lib/libmvec.so.1
-usr/lib/libncurses.so.6
 usr/lib/libncursesw.so.6
-usr/lib/libncursesw.so.6.0
-usr/lib/libnsl-2.26.so
-usr/lib/libnsl.so
+usr/lib/libncursesw.so.6.1
+usr/lib/libnsl-2.28.so
 usr/lib/libnsl.so.1
 usr/lib/libnspr4.so
 usr/lib/libnss3.so
-usr/lib/libnss_compat-2.26.so
+usr/lib/libnss_compat-2.28.so
 usr/lib/libnss_compat.so
 usr/lib/libnss_compat.so.2
-usr/lib/libnss_db-2.26.so
+usr/lib/libnss_db-2.28.so
 usr/lib/libnss_db.so
 usr/lib/libnss_db.so.2
-usr/lib/libnss_dns-2.26.so
+usr/lib/libnss_dns-2.28.so
 usr/lib/libnss_dns.so
 usr/lib/libnss_dns.so.2
-usr/lib/libnss_files-2.26.so
+usr/lib/libnss_files-2.28.so
 usr/lib/libnss_files.so
 usr/lib/libnss_files.so.2
-usr/lib/libnss_hesiod-2.26.so
+usr/lib/libnss_hesiod-2.28.so
 usr/lib/libnss_hesiod.so
 usr/lib/libnss_hesiod.so.2
-usr/lib/libnss_nis-2.26.so
-usr/lib/libnss_nisplus-2.26.so
-usr/lib/libnss_nisplus.so
-usr/lib/libnss_nisplus.so.2
-usr/lib/libnss_nis.so
-usr/lib/libnss_nis.so.2
 usr/lib/libnssutil3.so
 usr/lib/libpanelw.so.6
-usr/lib/libpanelw.so.6.0
+usr/lib/libpanelw.so.6.1
 usr/lib/libplc4.so
 usr/lib/libplds4.so
 usr/lib/libpopt.so.0
 usr/lib/libpopt.so.0.0.0
-usr/lib/libpthread-2.26.so
+usr/lib/libpthread-2.28.so
 usr/lib/libpthread.so
 usr/lib/libpthread.so.0
 usr/lib/libreadline.so.7
 usr/lib/libreadline.so.7.0
-usr/lib/libresolv-2.26.so
+usr/lib/libresolv-2.28.so
 usr/lib/libresolv.so
 usr/lib/libresolv.so.2
-usr/lib/librpmio.so.7
-usr/lib/librpmio.so.7.0.1
-usr/lib/librpm.so.7
-usr/lib/librpm.so.7.0.1
-usr/lib/librt-2.26.so
+usr/lib/librpmio.so.8
+usr/lib/librpmio.so.8.1.0
+usr/lib/librpm.so.8
+usr/lib/librpm.so.8.1.0
+usr/lib/librt-2.28.so
 usr/lib/librt.so
 usr/lib/librt.so.1
 usr/lib/libSegFault.so
@@ -808,7 +797,7 @@ usr/lib/libtdnf.so.0.0.0
 usr/lib/libthread_db-1.0.so
 usr/lib/libthread_db.so
 usr/lib/libthread_db.so.1
-usr/lib/libutil-2.26.so
+usr/lib/libutil-2.28.so
 usr/lib/libutil.so
 usr/lib/libutil.so.1
 usr/lib/libverto.so
@@ -861,6 +850,8 @@ usr/lib/rpm/platform/armv5tejl-linux/
 usr/lib/rpm/platform/armv5tejl-linux/macros
 usr/lib/rpm/platform/armv5tel-linux/
 usr/lib/rpm/platform/armv5tel-linux/macros
+usr/lib/rpm/platform/armv5tl-linux/
+usr/lib/rpm/platform/armv5tl-linux/macros
 usr/lib/rpm/platform/armv6hl-linux/
 usr/lib/rpm/platform/armv6hl-linux/macros
 usr/lib/rpm/platform/armv6l-linux/
@@ -1045,9 +1036,9 @@ usr/share/doc/bash/NEWS
 usr/share/doc/bash/POSIX
 usr/share/doc/bash/RBASH
 usr/share/doc/bash/README
-usr/share/doc/toybox-0.7.3/
-usr/share/doc/toybox-0.7.3/LICENSE
-usr/share/doc/toybox-0.7.3/README
+usr/share/doc/toybox-0.7.7/
+usr/share/doc/toybox-0.7.7/LICENSE
+usr/share/doc/toybox-0.7.7/README
 usr/share/i18n/
 usr/share/i18n/charmaps/
 usr/share/i18n/charmaps/ISO-8859-1.gz
```

</details>

File: library/photon [hunk 1]
```diff
@@ -12,6 +12,13 @@ Tags: 1.0, 1.0-20181017
 GitFetch: refs/heads/1.0-20181017
 GitCommit: 46235eaadf681be9e5277efd8bc694bf0cb19510
 
+Tags: 3.0, 3.0-20181030
```
Comment (yosifkit) [inline library/photon]: Did you want to move the `latest` tag  from `2.0` down to `3.0`?

Comment (YustasSwamp) [reply library/photon]: @yosifkit Thanks for attention. We will move `latest` to `3.0` later. Current 3.0 is in beta state.

**Comment by yosifkit** [comment]
Build test of #5017; e77fd768; `amd64` (`photon`):

```console

$ bashbrew build photon:2.0
Building bashbrew/cache:b6777b47e23976818eabfc085e35bea46ff7542d5c565e4e40f1aa52ffcf279e (photon:2.0)
Tagging photon:2.0
Tagging photon:2.0-20181017
Tagging photon:latest

$ test/run.sh photon:2.0
testing photon:2.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build photon:1.0
Building bashbrew/cache:0a8077984dee60a7e0a1b4cb7328ab8296f0caf5bef498481bddc9d098690a88 (photon:1.0)
Tagging photon:1.0
Tagging photon:1.0-20181017

$ test/run.sh photon:1.0
testing photon:1.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build photon:3.0
Building bashbrew/cache:7b5ef780b386b07cfd29b2d027733a272b42321307a0853f98ccd85f656e16ee (photon:3.0)
Tagging photon:3.0
Tagging photon:3.0-20181030

$ test/run.sh photon:3.0
testing photon:3.0
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build photon:dev
Building bashbrew/cache:a1447ba8855b4d9f4db229e6acca297e5881318efd1ac40118c3c138000f7fb7 (photon:dev)
Tagging photon:dev
Tagging photon:dev-20180904

$ test/run.sh photon:dev
testing photon:dev
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...cut: /etc/shadow: No such file or directory
warning: garbage password detected for 'bin': 'x'
warning: garbage password detected for 'systemd-timesync': 'x'
warning: garbage password detected for 'systemd-resolve': 'x'
warning: garbage password detected for 'systemd-network': 'x'
warning: garbage password detected for 'messagebus': 'x'
warning: garbage password detected for 'daemon': 'x'
warning: garbage password detected for 'systemd-journal-upload': 'x'
warning: garbage password detected for 'nobody': 'x'
warning: garbage password detected for 'systemd-bus-proxy': 'x'
warning: garbage password detected for 'systemd-journal-remote': 'x'
warning: garbage password detected for 'systemd-journal-gateway': 'x'
passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**